README.md

```diff
@@ -1,13 +1,27 @@
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/comit-network/community)
+<a href="https://comit.network">
+<img src="logo.svg" height="120px">
+</a>
 
-# COMIT Javascript SDK
+---
 
-A software development kit for developing Javascript applications on top of the COMIT protocol reference implementation [comit-rs](https://github.com/comit-network/comit-rs).
+[COMIT](https://comit.network) is an open protocol facilitating cross-blockchain applications.
+With [COMIT](https://comit.network) you can for examples exchange Bitcoin for any Erc20 token directly with another person.
```

Comment:
English grammar + small improvement proposal (feel free to edit): 
```suggestion
For example, with [COMIT](https://comit.network) you can exchange Bitcoin for Ether or any ERC20 token directly with another person.
```

Reply:
FWIW, I like @bonomat's edit.